packages/components/src/Link/Link.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { PropsWithChildren } from "react";
+
+interface LinkProps {
+  readonly external?: boolean;
+  readonly url: string;
+}
+
+export function Link({
+  external = false,
+  url,
+  children,
+}: PropsWithChildren<LinkProps>) {
+  const target = external ? "_blank" : "_self";
+
```

Comment:
There is quite a bit inside the Atlantis Button component that might be worth considering in your initial implementation. I don't understand all of it, but it might queue up some good discussion of things we want to include out of the box for <Link>.

Reply:
Hmm, I looked through button - it does some fancy prop handling, but I don't think that's helpful here unless this component gets a lot more complex.  Interestingly, the button does render `<a>` elements when provided a URL, but it also still renders them to look like buttons, so while semantically questionable, it doesn't seem to be an overlap.

That said, from what I see with the Button, it looks like adding client routing support will be really easy!

Reply:
From button, I'm thinking we might want to include these props just to support the same expressibility of aria attributes:
 ```
    "aria-controls": ariaControls,
    "aria-haspopup": ariaHaspopup,
    "aria-expanded": ariaExpanded,
```    
e.g. docs:    https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

Reply:
I'd looked through those: they aren't intended for use on <a> (or links in general).  It's a definite anti-pattern / misuse of the component if these did end up being relevant to a use case I think.

Reply:
Hmm, curious where you see that. IMHO, these can be on links that trigger popups or control or expand other content in the DOM. This section of [aria-expanded](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#associated_roles) lists it as relevant for anything with a `link` role, which an <a> automatically has. 
This article has an example that uses one of them on a link. https://www.digitala11y.com/aria-haspopup-properties/

We can continue the discussion/gather other input on the implementation PR though!